models/refresh_token.go

```diff
@@ -100,6 +100,18 @@ func GetValidChildToken(tx *storage.Connection, token *RefreshToken) (*RefreshTo
 	return refreshToken, nil
 }
 
+func FindTokenBySessionID(tx *storage.Connection, sessionId *uuid.UUID) (*RefreshToken, error) {
+	refreshToken := &RefreshToken{}
+	err := tx.Q().Where("session_id = ?", sessionId).First(refreshToken)
```

Comment:
this has to be the latest unrevoked token in the session right? i don't think `First` guarantees that

Reply:
Yup, good catch!  Will add an `order_by` and a check for `revoked=false` which should guarantee that it's the latest refresh token